internal/graph/check.go

```diff
@@ -197,6 +199,10 @@ func union(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHandle
 		}
 	}
 
+	if err != nil {
+		return &openfgapb.CheckResponse{Allowed: false}, err
+	}
+
 	return &openfgapb.CheckResponse{Allowed: false}, nil
```

Comment:
```suggestion
	return &openfgapb.CheckResponse{Allowed: false}, err
```
The check for err != nil is redundant.

Reply:
✅ done